static/js/components/PublishDrawer.test.tsx

```diff
@@ -1,27 +1,36 @@
+import React from "react"
 import moment from "moment"
 import sinon, { SinonStub } from "sinon"
 import { act } from "react-dom/test-utils"
 import { isEmpty } from "ramda"
 
 import { siteApiActionUrl, siteApiDetailUrl } from "../lib/urls"
-import { shouldIf } from "../test_util"
+import { assertInstanceOf, shouldIf } from "../test_util"
 import { makeWebsiteDetail } from "../util/factories/websites"
-import IntegrationTestHelper, {
+import IntegrationTestHelperOld, {
```

Comment:
Is there something that `IntegrationTestHelperOld` can test that the new one can't?

Reply:
No. But the old one uses Enzyme, and the new one uses `@testing-library/react` ("RTL") (and other packges in the `@testing-library` family). RTL has a somewhat different API than Enzyme, so converting tests will be a bit of effort. 

I've been trying to write new tests with RTL + the new helper, and have been converting old tests when refactoring significantly. (Which is natural, since the Enzyme tests tend to be a bit brittle, so refactoring tends to break them).